common/agent.py

```diff
@@ -19,9 +19,10 @@ def ShowConfig(config):
 
 def UpdateConfig(config, file_path, key_name='algo'):
     _config = load_config(file_path)
+    values = _config[key_name]
     try:
-        for key in _config[key_name]:
-            config[key] = _config[key]
+        for key in values:
+            config[key] = values[key]
```

Comment:
refuse

Reply:
The current code leads to an error, since the configuration dictionary now contains section keys, rather than parameter names:
``` algo:
   actor_lr: 0.0005
   batch_size: 1024
   beta: 0.001
   critic_lr: 0.001
   epoch: 4
   gamma: 0.99
```